public/css/logixRelease.css

```diff
@@ -109,4 +109,10 @@ a.btn.btn-outline-danger {
     margin-top: 10px;
 }
 
+#error_explanation{
+    color: crimson;
+    background: #cea0a5;
```

Comment:
Do we really need this `backgroud` property set? What happens if you remove it?

Reply:
I thought #f0f0f0 may be good
What do you suggest?